res/layout/list_fragment.xml

```diff
@@ -40,19 +48,14 @@
         android:layout_height="match_parent"
         android:visibility="gone" >
 
-        <ScrollView
```

Comment:
This is also working in my version.
The only problem is, that it cannot display "nothing in here..."